modules/dxp/apps/segments/segments-experiment-web/test/js/components/ClickGoalPicker.es.js

```diff
@@ -31,14 +31,16 @@ describe('Segments Experiments with Click Goal', () => {
 		};
 
 		const {getByText} = renderApp({
-			initialSegmentsExperiment: segmentsExperiment,
+			initialSegmentsExperiment: experiment,
 		});
 
-		getByText(experiment.name);
-		getByText('review-and-run-test');
+		expect(getByText(experiment.name)).toBeInTheDocument();
+		expect(getByText('review-and-run-test')).toBeInTheDocument();
+		expect(getByText('click-goal-description')).toBeInTheDocument();
+		expect(getByText('element-id')).toBeInTheDocument();
 	});
 
-	it('User clicks in set element button, the button turns from default to active mode', () => {
+	it('User clicks in select clickable element button, the button turns from default to active mode', () => {
```

Comment:
Can we make this more readable by putting quotes around "select clickable element" (or something like that?).

I also notice that in the UI it's actually "Select Clickable Element", with caps, so maybe we should match that here.

Reply:
Sure!